### PR TITLE
Move to Razzle - SSR ALL THE THINGS

### PR DESCRIPTION
**Pull request by SaraVieira**
Things that are still broken:

- Eslint is being a lil bitch
- Lazyload
- white screen when main page is loaded

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,67 +4,78 @@
...
+    "isomorphic-fetch": "^2.2.1",
     "lodash.debounce": "^4.0.8",
-    "preact": "8.2.5",
-    "preact-async-route": "^2.2.0",
-    "preact-compat": "3.17.0",
-    "preact-portal": "^1.1.3",
-    "preact-router": "^2.6.0",
     "random-id": "^0.0.2",
+    "razzle": "^2.0.0-alpha.12",
+    "react": "^16.4.0",
```
Comment (pksjce) [inline package.json]: yay :) I was really feeling out of depth with preact stuff!

**Comment by SaraVieira** [comment]
SOOOO I have made it to be server rendered and I could really use people input in this PR

Netlify doesn't work anymore because the code needs a server now but I can add the same thing but with now for the PR

I have merged everyone's changes into here and let me know if something isn't working


What did this do:
 - The website is instant almost now
 - Removed shuffled talks
 - All the search and pagination are done on the server so I don't load a bunch of stuff at the start
- Removal of a lot of libraries
- Change to react and razzle


Here is a demo: https://awesome-talks-aowifkamru.now.sh

**Comment by SaraVieira** [comment]
@hosmelq @varjmes @cain @tsiq-swyx

**Comment by hosmelq** [comment]
I see that the search icon is jumping, I think it's because the CSS does not load in the first render. I'll take a look.

**Comment by hosmelq** [comment]
Ok, i think this solve the problem. https://fontawesome.com/how-to-use/server-side-rendering#css

**Comment by cain** [comment]
@SaraVieira #40

**Comment by tsiq-swyx** [comment]
this is a philosophical/architectural question i struggle with right now. by default i start with gatsby for all my projects so that i dont need a server. not saying i know if i need SSR or anything but i just havent needed afterjs/razzle/nextjs since using gatsby. 🤷‍♂️ i dont super care do whatever works :) but thats my 2 cents since u asked.

**Comment by SaraVieira** [comment]
Thank you all!

@tsiq-swyx I get that, I have never used Gatsby extensively but I want to point out the reason I want a server is to cache stuff from graphcms otherwise I may go overboard on their plan and pay a bunch per month and stuff like speakers is always the same so it can totally be cached in the backend :)

Hope this makes sense and here is a Pic of the thing on graphcms after like 10days 
screenshot_20180530-070641

**Comment by SaraVieira** [comment]
I shall be merging this :D

**Comment by farskid** [comment]
@SaraVieira @hosmelq @cain @tsiq-swyx @pksjce Just dropped by to thank ya'll for the effort and also say that I love how fast and funny you commit @SaraVieira , lol

**Comment by SaraVieira** [comment]
Thank @farskid  :D

**Pull request merged**